src/chains/filecoin/filecoin/src/blockchain.ts

```diff
@@ -781,9 +790,6 @@ export default class Blockchain extends Emittery.Typed<
 
     const signature = await account.address.signProposal(proposal);
 
-    // TODO: I'm not sure if should pass in a hex string or the Buffer alone.
-    // I *think* it's the string, as that matches my understanding of the Go code.
-    // That said, node that Buffer vs. hex string returns a different CID...
```

Comment:
This was a TODO from @tcoulter's original code. We can keep it if you want it, but I don't plan on changing the implementation here since we're not ensuring cryptographic integrity. Up to yall! I can make an issue if you want to keep it